libs/platform/src/lib/components/upload-collection/upload-collection/upload-collection.component.ts

```diff
@@ -156,6 +157,13 @@ export class UploadCollectionComponent {
     @Input()
     showSearch = true;
 
+    /**
+     * Allows to use the same name for a file when editing the file name.
+     * 'Same name' refers to an already existing file name in the list.
+     * */
+    @Input()
+    sameFilenameAllowed = false;
```

Comment:
Not a big deal but in my opinion it should be named `sameFileNameAllowed` to be consistent with `_fileNameChange`